# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/tcp_wrappers/errno-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN tcp_wrappers_7.6-orig/hosts_access.c tcp_wrappers_7.6/hosts_access.c
--- tcp_wrappers_7.6-orig/hosts_access.c	1997-02-12 02:13:23.000000000 +0100
+++ tcp_wrappers_7.6/hosts_access.c	2003-03-05 16:07:28.000000000 +0100
@@ -35,7 +35,7 @@
 #include <string.h>
 
 extern char *fgets();
-extern int errno;
+/*extern int errno;*/
 
 #ifndef	INADDR_NONE
 #define	INADDR_NONE	(-1)		/* XXX should be 0xffffffff */
diff -urN tcp_wrappers_7.6-orig/inetcf.c tcp_wrappers_7.6/inetcf.c
--- tcp_wrappers_7.6-orig/inetcf.c	1997-02-12 02:13:24.000000000 +0100
+++ tcp_wrappers_7.6/inetcf.c	2003-03-05 16:07:41.000000000 +0100
@@ -15,7 +15,7 @@
 #include <errno.h>
 #include <string.h>
 
-extern int errno;
+/*extern int errno;*/
 extern void exit();
 
 #include "tcpd.h"
diff -urN tcp_wrappers_7.6-orig/percent_m.c tcp_wrappers_7.6/percent_m.c
--- tcp_wrappers_7.6-orig/percent_m.c	1994-12-28 17:42:37.000000000 +0100
+++ tcp_wrappers_7.6/percent_m.c	2003-03-05 16:07:58.000000000 +0100
@@ -12,7 +12,7 @@
 #include <errno.h>
 #include <string.h>
 
-extern int errno;
+/*extern int errno;*/
 #ifndef SYS_ERRLIST_DEFINED
 extern char *sys_errlist[];
 extern int sys_nerr;
diff -urN tcp_wrappers_7.6-orig/tcpdchk.c tcp_wrappers_7.6/tcpdchk.c
--- tcp_wrappers_7.6-orig/tcpdchk.c	1997-02-12 02:13:25.000000000 +0100
+++ tcp_wrappers_7.6/tcpdchk.c	2003-03-05 16:08:14.000000000 +0100
@@ -31,7 +31,7 @@
 #include <netdb.h>
 #include <string.h>
 
-extern int errno;
+/*extern int errno;*/
 extern void exit();
 extern int optind;
 extern char *optarg;
diff -urN tcp_wrappers_7.6-orig/tli-sequent.c tcp_wrappers_7.6/tli-sequent.c
--- tcp_wrappers_7.6-orig/tli-sequent.c	1994-12-28 17:42:51.000000000 +0100
+++ tcp_wrappers_7.6/tli-sequent.c	2003-03-05 16:08:32.000000000 +0100
@@ -30,7 +30,7 @@
 #include <errno.h>
 #include <string.h>
 
-extern int errno;
+/*extern int errno;*/
 extern char *sys_errlist[];
 extern int sys_nerr;
 extern int t_errno;
diff -urN tcp_wrappers_7.6-orig/tli.c tcp_wrappers_7.6/tli.c
--- tcp_wrappers_7.6-orig/tli.c	1997-03-21 19:27:26.000000000 +0100
+++ tcp_wrappers_7.6/tli.c	2003-03-05 16:08:45.000000000 +0100
@@ -39,7 +39,7 @@
 #include <string.h>
 
 extern char *nc_sperror();
-extern int errno;
+/*extern int errno;*/
 extern char *sys_errlist[];
 extern int sys_nerr;
 extern int t_errno;
diff -urN tcp_wrappers_7.6-orig/workarounds.c tcp_wrappers_7.6/workarounds.c
--- tcp_wrappers_7.6-orig/workarounds.c	1996-03-19 16:22:26.000000000 +0100
+++ tcp_wrappers_7.6/workarounds.c	2003-03-05 16:08:59.000000000 +0100
@@ -22,7 +22,7 @@
 #include <syslog.h>
 #include <string.h>
 
-extern int errno;
+/*extern int errno;*/
 
 #include "tcpd.h"
 
